tasks/base/general/configure_podman.yml

```diff
@@ -0,0 +1,74 @@
+---
+- name: Stop the podman service
+  systemd:
+    name: podman.service
+    state: stopped
+
+- name: Configure podmanapi.service
+  template:
+    src: podmanapi.service
+    dest: /usr/lib/systemd/user/podmanapi.service
+
+- name: alias docker and use socket
+  template:
+    src: docker.alias
+    dest: /usr/bin/docker
+    mode: 0755
+
+- name: set Temporary storage location
+  lineinfile:
+    path: /etc/containers/storage.conf
+    regexp: "^runroot = "
+    line: runroot = "/mnt/data/docker/runroot/"
+    backrefs: yes
+    create: yes
+
+- name: set Primary Read/Write location of container storage
+  lineinfile:
+    path: /etc/containers/storage.conf
+    regexp: "^graphroot = "
+    line: graphroot = "/mnt/data/docker/graphroot/"
+    backrefs: yes
+    create: yes
+
+- name: set location of container storage for rootless
+  lineinfile:
+    path: /etc/containers/storage.conf
+    insertafter: '^# rootless_storage_path = '
+    line: rootless_storage_path = "/mnt/data/docker/rootless/$UID/"
+    create: yes
+
+- name: set overlay2 storage driver
```

Comment:
This is a little confusing since we configure overlayfs in [vars/os_RedHat_8.yml](https://github.com/elastic/ansible-elastic-cloud-enterprise/pull/148/files#diff-732e5a8fceff74236a8d28b561900d67de049da4eb0acaa916b7c5039ffef100)

Is this intentional?

Reply:
This is half an error: I wanted to set overlay2, but I discovered issues where podman is sometime (why? when?) expecting paths of the blobs containing `overlay2` while it was well stored in an other path containing `overlay`. But I should clean this when the PR will be ready (that should have been draft)